src/App.css

```diff
@@ -73,6 +73,10 @@ body {
   border: 1px solid white;
 }
 
+.btn:hover {
+  background-color: darkgrey;
+}
+
 .btn-orange {
   color: white;
   background-color: coral;
```

Comment:
- [ ] [OPTIONAL] You have done an amazing job with this implementation but I think giving more keen attention to user experience and accessibility in terms of the size of your buttons and the position of the `Calculator` would be nice. Achieving a centralized calculator might make more use of the extra spaces without seeming off.

![image](https://github.com/tawakuliKH/react-math-magicians/assets/99715640/d6099e6d-90c8-4518-82cf-5d1461e0ea46)